Autopilot/Inc/batterysensor.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Battery Sensor Header File
+ * 
+ * This will basically sense the voltage of the battery.
+ * 
+ * Author: Sahil Kale
+ *  
+ **/
+
+#ifndef BATTERYSENSOR_HPP
+#define BATTERYSENSOR_HPP
+
+/**
+ * Starts the ADC for the battery sensor 
+ */
+virtual void initBatterySensor();
+
+/**
+ *  Returns the battery voltage as a float
+ * 
+ * @param battery The selected battery to return the voltage from
+ */
+virtual float getBatteryVoltage(int battery);
```

Comment:
I could be wrong, but is voltage the only thing wer'e gathering ?

Reply:
Yup, although it's for 2 batteries apparently so I just specified as an int